prql-js/README.md

```diff
@@ -25,11 +25,36 @@ function rq_to_sql(rq_json: string): string;
 
 ### From NodeJS
 
+Direct usage
+
 ```javascript
-const prql = require("prql-js");
+const prqljs = require("prql-js");
 
-const sql = compile(`from employees | select first_name`);
-console.log(sql);
+const sql = prqljs.compile(`from employees | select first_name`);
+console.log(sql.sql);
+```
+
+Template literal
+
+```javascript
+const prqljs = require("prql-js");
+const prql = (string) => prqljs.compile(string[0] || "");
```

Comment:
I really like that you've just provided an example for template function instead of trying to squeeze it into our package that generated by wasm-pack anyway.